boefjes/boefjes/job_handler.py

```diff
@@ -23,6 +23,8 @@
 from octopoes.models.exception import ObjectNotFoundException
 from octopoes.models.types import OOIType
 
+MIMETYPE_MIN_LENGTH = 5  # two chars before, and 2 chars after the slash ought to be reasonable
```

Comment:
Although the suggested implementation works and seems as a practical approach, I was just curious if this is the right approach to validate MIME types. Why not just validate against the format `type/subtype;parameter=value`? A MIME type always has both a type and a subtype, never just one or the other, and an optional parameter can be added to provide additional details.

In my view a MIME type in the format `<two characters> / <two characters>` is not a more valid or reasonable MIME type than a non-existing or defined (with valid format) MIME type or a bogus one (e.g. `ab/cd`)